src/cacheHttpClient.ts

```diff
@@ -1,26 +1,29 @@
 import * as core from "@actions/core";
 import * as fs from "fs";
-import { BearerCredentialHandler } from "typed-rest-client/Handlers";
-import { HttpClient, HttpCodes } from "typed-rest-client/HttpClient";
-import { IHttpClientResponse } from "typed-rest-client/Interfaces";
+import { BearerCredentialHandler } from "@actions/http-client/auth";
+import { HttpClient, HttpCodes, ITypedResponse } from "@actions/http-client";
```

Comment:
@bryanmacfarlane A few comments around http-client 1.0.4 1

1: We have other interfaces exported in `@actions/http-client/interfaces` but the new `ITypedResponse` comes directly from `@actions/http-client`

Reply:
Yeah, I did that intentionally.  The other response interface is off the http client directly.  https://github.com/actions/http-client/blob/master/index.ts#L53

Reply:
oops - no, you're right.  that other one was a class not an interface